### ISSUE TITLE
Selecting packages interactively is no fun at all

### ISSUE THREAD

**Issue by kakulukia** [python-poetry/poetry#528]
- [x] I have searched the [issues](https://github.com/sdispater/poetry/issues) of this repo and believe that this is not a duplicate.

## Issue

While executing `poetry init` i get asked to interactively define my dependencies. This feature is very welcome at this point, BUT the way it works is a little suboptimal. While `poetry add` never made the wrong decision for me so far,  in this interactive mode after wanting to install Django i get a list of 100 search results not even containing the actual Django package.
Poetry never tries to just guess and install the right packages like it does on `poetry add`. 

Why?
Adding Django (and any other package) just works, but interactively selecting is no fun!

Please make it fun again and just install the best guess!
Or to be more save at least display the best guess (as it would be resolved on add) and just let the user confirm and maybe select an alternative.

**Comment by sdispater** [comment]
That's an issue that has been reported before and while I agree it's not optimal that's the way the current XML-RPC API works so the improvements must be done by Poetry.

A first improvement can be that if an exact match is found it's returned first and we can also limit the number of results returned.

**Comment by Woile** [comment]
Hey! I was wondering, does it need to be part of `init`? so far in my experience I've avoided using it always. And the few times I miss typed yes, it happened the same situation described by kakulukia. 

Hope the feedback is useful, I love the tool so far!

**Comment by kakulukia** [comment]
I also wouldnt mind if that was removed from the init process. Good idea!

**Comment by pmav99** [comment]
@Woile @kakulukia you can always run `poetry init -n` which will create an `pyproject.toml` with the default values and no dependencies. You can then add whatever you need by `poetry add <package>`.

**Comment by harrybiddle** [comment]
+1. I used this for the first time and found it to be a pretty poor experience. I'm in favour of removing it, or least making it not default (#179).

When I typed `psycopg2`, instead of installing it it gives me a massive list of that library and similarly named ones. So long, that the option I actually want requires me to scroll up:

<img width="264" alt="grafik" src="https://user-images.githubusercontent.com/5887227/58696590-29bc8880-8398-11e9-891a-8a504fa29d78.png">

Also, if I slightly mistype the library I get a completely different one:


```bash
Search for a package: pyscopg2
Found 1 packages matching pyscopg2

Enter package # to add, or the complete package name if it is not listed:
 [0] sanepg
 >
```

Even if `poetry init -n` is a workaround, a bad experience shouldn't be the default for newcomers to Poetry.
